### DISCUSSION TITLE
audio echo issues.. also posted as an issue.. but looking for any input.

### DISCUSSION THREAD

**Discussion by CubeXR** [mozilla/hubs#3929] [General Questions & Discussion]
we have OBS recorded many show from HUBS in the months in the past. 

as of the new interface update, we keep getting echos of voices as soon as more than one avatar in in world. speaking or whatever. 

we tried all settings and everything we had done in the past, even headphones, mics, etc vs HMDs mics. webcam mics etc. whatever. the echoing comes in and out with no rhythm or reason. different people each time. different people hearing it from others at different times... had people on higher end PCs.. macbook pros, chromebooks, and PC laptops.. all have done this before.. taped mediaverse lounges etc.. we cant get good audio ... we are now forced to record video in hubs.. and try to use separate mics to record into a third disc based app. while still using OBS to record the video and get a useless audio track... any ideas.. 

also wasnt there used to be a way to remove "3d spatial audio" from the worlds. and force all audio to the same flat volume. so all avatars record at the audio levels to the OBS camera avatar? 

thanks for any input. memory leaks ? latency issues? we are all stumped. just people trying to use HUBS for a virtual tv studio ;) simple right?;)

**Comment by robinkwilson** [comment]
Interesting we have not heard of other reports of this. 

Can you link the issue you posted as well with the reproduction steps?

Does this only happen when you start recording with OBS?
